multi/stm32l4-multi/flash.c

```diff
@@ -233,6 +259,54 @@ size_t flash_writeData(uint32_t offset, const char *buff, size_t size)
 }
 
 
+size_t flash_writeOtp(uint32_t offset, const char *buff, size_t size)
+{
+	size_t ret, len;
+	size_t pos = 0;
+	uint32_t temp[2];
+	volatile uint32_t *ptr = (void *)offset;
```

Comment:
Why we're expecting user to know OTP area address? 
```suggestion
	volatile uint32_t *ptr = (void *)(offset + OTP_ADDR);
```

Reply:
Done. Now user just pass offset.